### PR TITLE
MTA-525 Disconnected install in RN

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves:
https://issues.redhat.com/browse/MTA-525
https://issues.redhat.com/browse/MTA-18

MTA-6.1.0
Preview:
https://deploy-preview-683--windup-documentation.netlify.app/docs/release-notes/master/index.html#mta-rn-new-features-6-1-0_release-notes
https://deploy-preview-683--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#installing-mta-operator-in-disconnected-environment_user-interface-guide

### CODE REVIEW

**Review by ibragins — APPROVED** [review]
LGTM

**Comment by midays** [comment]
@sbeskin-redhat Thank you for putting this together, 
I just have a question; are we assuming that customers already know what a disconnected cluster is and what are the restrictions of it? e.g. limited access to external resources, difficulty in pulling container images, limited availability of software updates, ... etc

**Comment by sbeskin-redhat** [comment]
@midays 
Yes, i think we assume that the user knows what a disconnected cluster or environment is. It appears in OpenShift documentation.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 1]
```diff
@@ -161,6 +161,34 @@ spec:
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
 
+[id="installing-mta-operator-in-disconnected-environment_{context}"]
+== Installing the {ProductName} Operator in a disconnected {ocp-short} environment
+
```
Comment (RichardHoch) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:166]: This really should be a separate module, and probably a different section as we have for MTC https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/migration_toolkit_for_containers/index#installing-mtc-restricted

It is a separate procedure, so I suggest changing line 169 to:

.Procedure

* Configure ...

**Comment by RichardHoch** [comment]
@sbeskin-redhat @anarnold97 Given the time issue, I'm calling this LGTM but recommending a small tweak now and a doc improvement as described above.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc:166]: @RichardHoch 
Re line 169: It's not really a procedure, it's just an addition to the procedure referenced in line 167.

Comment (sbeskin-redhat) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc:166]: @RichardHoch  Tweaked, thank you

**Pull request merged**